src/main/java/net/pms/dlna/DLNAResource.java

```diff
@@ -1928,10 +1936,11 @@ private String getDlnaOrgOpFlags(RendererConfiguration mediaRenderer) {
 	 * @return String representation of the DLNA.ORG_PN flags
 	 */
 	@SuppressWarnings("deprecation")
-	private String getDlnaOrgPnFlags(RendererConfiguration mediaRenderer, int localizationValue) {
+	private String getDlnaOrgPnFlags(RendererConfiguration mediaRenderer, int localization) {
```

Comment:
Why rename this parameter here and then reassign it to the old name on line 1943?